packages/webapp/src/app/layout.tsx

```diff
@@ -11,7 +12,9 @@ export const metadata = {
 export default function RootLayout({ children }: { children: React.ReactNode }) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
+        <PageBase>{children}</PageBase>
```

Comment:
The `PageBase` (better name ideas?) allows for a uniform structure (e.g. header + theme) for all pages.

Reply:
You can also just put those things directly into the `RootLayout` component here (I am not sure you need the extra layer of indirection).

Reply:
It kept complaining. I had to add `use react` at the top and when I did that it was complaining about having an `export` in a file that has `use react`.